content/blog/vs-firefox-relay.md

```diff
@@ -67,6 +67,24 @@ Firefox Relay relies on Amazon SES to receive and forward emails to your mailbox
 
 SimpleLogin doesn't rely on any external service. It is cloud agnostic and can be run anywhere, either on a small home server for people who want self host or on a powerful dedicated server. We did use Amazon for SimpleLogin first version but [decided to leave the platform in February 2020](/blog/we-left-aws/).
 
+{{< rawhtml >}}
+<br>
+  <div style="padding: 35px; background-color: #f8f9fa; border-radius: 15px;">
+    <p class="mb-2 h5"><strong>Try SimpleLogin for free</strong></p>
+    <p class="mb-2">
+        Still hesitant on the best option to choose to <strong>mask your email</strong> address between Firefox Relay and SimepleLogin. SimpleLogin has a generous freemium plan that allows you to use up to <strong>10 aliases for free</strong>.
```

Comment:
I would change "Still hesitant on the best option to choose to" -> "Still hesitant on how to" and the question mark `?`